src/components/Pagination/Pagination.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useMemo } from 'react';
+import React, { useMemo, useRef, useState, useEffect } from 'react';
```

Comment:
Hey @ajadams! Thanks for the PR.  It looks like `useEffect` isn't actually used here.  Can you remove it?

```suggestion
import React, { useMemo, useRef, useState } from 'react';
```